src/main/java/evaluation/listeners/IGameListener.java

```diff
@@ -26,10 +31,9 @@ public interface IGameListener {
      * <p>
      * This is useful for Listeners that are just interested in aggregate data across many runs
      */
-    void allGamesFinished();
-
-    default boolean setOutputDirectory(String out, String time, String players) {return true;}
```

Comment:
I think this (And other stuff) is from the slightly order metric system...just be aware that the newer version is now in master, so see what breaks when you merge that in,